controllers/resource/template.go

```diff
@@ -179,7 +179,7 @@ func (r *CloudStackTemplate) TemplateResources(ctx context.Context, eksaCluster
 	}
 	clusterSpec.CloudStackDatacenter = &csdc
 	// control plane and etcd updates are prohibited in controller so those specs should not change
-	templateBuilder := cloudstack.NewCloudStackTemplateBuilder(&csdc.Spec, &cpCsmc.Spec, &etcdCsmc.Spec, workerNodeGroupMachineSpecs, r.now)
+	templateBuilder := cloudstack.NewTemplateBuilder(&csdc.Spec, &cpCsmc.Spec, &etcdCsmc.Spec, workerNodeGroupMachineSpecs, r.now)
```

Comment:
Renamed to avoid stuttering when used by other packages

Reply:
I don't think we need rename, maybe just follow the naming convention across all providers.

Reply:
actually, the linter was the one that forced this change.